docs/connector-development/config-based/connector-builder-compatibility.md

```diff
@@ -0,0 +1,210 @@
+# Connector Builder Compatibility Guide
+Answer the following questions to determine whether the Connector Builder is the right tool to build the connector you need:
+- [ ] [Is it a REST API returning a collection of records synchronously?](#is-the-integration-a-rest-api-returning-a-collection-of-records-synchronously)
+- [ ] [Is the API using one of the following authentication mechanism?](#what-type-of-authentication-is-required)
+    - [Basic HTTP](#basic-http)
+    - [API key injected in request header or query parameter](#api-key)
+    - [OAuth2.0 with long-lived refresh token](#is-the-oauth-refresh-token-long-lived)
+    - [Session authentication](#session)
+- [ ] [Is the data returned as JSON format?](#is-the-data-returned-as-json-format)
+- [ ] [If records are paginated, are they using one of the following mechanism?](#how-are-records-paginated)
+    - [Limit-offset](#limit-offset--offsetincrement-)
+    - [Page count](#page-count)
+    - [Link to the next page](#link-to-next-page--cursorpagination-)
+- [ ] [Do endpoints only require query parameters that are key-value pairs?](#does-the-integration-require-query-params-that-are-not-key-value-pairs)
+
+You can follow [this chart](https://whimsical.com/how-should-i-build-my-connector-YNXaNQhRuPuWy68Xg8Akhy) to guide you through the process.
```

Comment:
SHould we add a hint to this sentence that there is more nuance to what's supported and what not and that the chart explains those? From the context I expected that it would only guide through the 6 questions above.

Reply:
I updated the whimsical link to point to a board in our shared workspace that does not included the more complicated questions